plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlClient.java

```diff
@@ -470,6 +531,35 @@ public Optional<ColumnMapping> toColumnMapping(ConnectorSession session, Connect
         return Optional.empty();
     }
 
+    private static ColumnMapping mySqlDefaultVarcharColumnMapping(int columnSize, Optional<CaseSensitivity> caseSensitivity)
+    {
+        if (columnSize > VarcharType.MAX_LENGTH) {
+            return mySqlVarcharColumnMapping(createUnboundedVarcharType(), caseSensitivity);
+        }
+        return mySqlVarcharColumnMapping(createVarcharType(columnSize), caseSensitivity);
+    }
+
+    private static ColumnMapping mySqlVarcharColumnMapping(VarcharType varcharType, Optional<CaseSensitivity> caseSensitivity)
+    {
+        PredicatePushdownController pushdownController = caseSensitivity.orElse(CASE_INSENSITIVE) == CASE_SENSITIVE ? MYSQL_CHARACTER_PUSHDOWN : CASE_INSENSITIVE_CHARACTER_PUSHDOWN;
```

Comment:
Can we move the arguments to a new line ?

Reply:
refactored